bindata/etcd/pod.yaml

```diff
@@ -23,6 +23,11 @@ spec:
 
           cp /etc/kubernetes/static-pod-certs/secrets/etcd-all-peer/etcd-peer-NODE_NAME.crt /etc/kubernetes/etcd-backup-dir/system:etcd-peer-NODE_NAME.crt
           cp /etc/kubernetes/static-pod-certs/secrets/etcd-all-peer/etcd-peer-NODE_NAME.key /etc/kubernetes/etcd-backup-dir/system:etcd-peer-NODE_NAME.key
+          rm -f $(grep -l '^### Created by cluster-etcd-operator' /usr/local/bin/*)
+          ln -s /etc/kubernetes/static-pod-resources/etcd-certs/configmaps/etcd-scripts/etcd-member-remove.sh /usr/local/bin
+          ln -s /etc/kubernetes/static-pod-resources/etcd-certs/configmaps/etcd-scripts/etcd-snapshot-backup.sh /usr/local/bin
+          ln -s /etc/kubernetes/static-pod-resources/etcd-certs/configmaps/etcd-scripts/etcd-snapshot-restore.sh /usr/local/bin
```

Comment:
do we really want to maintain this list? could we use find to symlink all bash.sh files?

Reply:
We could do that too.